vaccine_feed_ingest/runners/tx/arcgis/normalize.py

```diff
@@ -0,0 +1,229 @@
+    """
+    Attempt to extract the city from the address. First use the word immediately preceding ", TX" if present. As a
+    secondary fallback, return the word immediately preceding a 5 digit zipcode
+    """
+    if not address:
+        return None
+    match = re.search(r"([\S]*),\s?TX", address, flags=re.IGNORECASE)
+    if match:
+        return match.group(1)
+    match = re.search(r"([\S]*)\s[0-9]{5}", address, flags=re.IGNORECASE)
+    return match.group(1) if match else None
+
+
+def _get_address(site_attributes: dict) -> Optional[schema.Address]:
+    address = site_attributes["ADDRESS"]
+    street = (
+        site_attributes["STREET"]
+        if site_attributes["STREET"]
+        else _get_fallback_street(address)
+    )
+    city = (
+        site_attributes["CITY"]
+        if site_attributes["CITY"]
+        else _get_fallback_city(address)
+    )
+    zipcode = (
+        site_attributes["ZIP"] if site_attributes["ZIP"] else _get_fallback_zip(address)
+    )
+    try:
+        return schema.Address(
+            street1=street,
+            street2=None,
+            city=city,
+            state="TX",
+            zip=zipcode,
+        )
+    except ValidationError as e:
+        logging.error(
+            f"Unable to parse address: "
+            f"address={site_attributes['ADDRESS']}, "
+            f"street={site_attributes['STREET']}, "
+            f"city={site_attributes['CITY']}, "
+            f"zip={site_attributes['ZIP']}, "
+            f"e={e}"
+        )
+        return None
+
+
+def _get_contacts(site_attributes: dict) -> Optional[List[schema.Contact]]:
+    contacts = []
+    public_phone = site_attributes["PublicPhone"]
+    if public_phone:
+        source_phone = re.sub("[^0-9]", "", public_phone)
+        if len(source_phone) == 11:
+            source_phone = source_phone[1:]
+        if len(source_phone) == 10:
+            phone = f"({source_phone[0:3]}) {source_phone[3:6]}-{source_phone[6:]}"
+            contacts.append(schema.Contact(phone=phone))
+        else:
+            logging.warning(f"Phone Number is unexpected size:{public_phone}")
+            contacts.append(schema.Contact(other=public_phone))
+
+    if site_attributes["WEBSITE"]:
+        contacts.append(schema.Contact(website=site_attributes["WEBSITE"]))
+    return contacts if contacts else None
+
+
+def _get_inventory(site_attributes: dict) -> List[schema.Vaccine]:
+    return [
+        schema.Vaccine(vaccine="jj", supply_level=site_attributes["JJ_AVAILABLE"]),
+        schema.Vaccine(
+            vaccine="pfizer", supply_level=site_attributes["PFIZER_AVAILABLE"]
+        ),
+        schema.Vaccine(
+            vaccine="pfizer_2", supply_level=site_attributes["PFIZER_AVAILABLE2"]
+        ),
+        schema.Vaccine(
+            vaccine="moderna", supply_level=site_attributes["MODERNA_AVAILABLE"]
+        ),
+        schema.Vaccine(
+            vaccine="moderna_2", supply_level=site_attributes["MODERNA_AVAILABLE2"]
+        ),
```

Comment:
Could you update the vaccines to normalize to `pfizer`, `moderna`, or `janssen ` (ignore the `_2` if it's provided)?

Also - where might be a good place in our documentation for me to note this request (I know it's not written anywhere at the moment, and I don't want to surprise you with it in a PR review)?

Reply:
a note near the relevant part of the normalizing/schema section of the wiki might be nice

Reply:
[Added](https://github.com/CAVaccineInventory/vaccine-feed-ingest/wiki/Runner-pipeline-stages#normalize)!

Reply:
Ok I represented these as the sum (e.g. moderna + moderna_2). Let me know if it should be only the first doses.
https://github.com/CAVaccineInventory/vaccine-feed-ingest/pull/134/commits/1bf88f731d68818a3af3242079fa2f5c2ee6c292